charts/cloud-connector/README.tpl

```diff
@@ -14,92 +14,85 @@ $ pre-commit run -a
 
 # Cloud Connector
 
-[{{ .Project.Name }}]({{ .Project.URL }}) - {{ .Project.Description }}
```

Comment:
please use existing variables for the short intro, then you can add whatever required additionally

Reply:
What's the purpose of short description

Reply:
using variables like {{ .Project.Name }}]({{ .Project.URL }}) - {{ .Project.Description }} in the tplfile,  as its done in the whole repository for other charts too.
resoled offline